### PR TITLE
Update runtest.cmd help to reflect reality

### PR DESCRIPTION
**Pull request by BruceForstall**

### CODE REVIEW

**Comment by BruceForstall** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test

**Comment by BruceForstall** [comment]
@RussKeldorph @jashook Can someone take a look?

**Comment by jashook** [comment]
Lgtm minus one nit, it would be nice to align the lines from the hyphen on.

/cc @wtgodbe

**Comment by BruceForstall** [comment]
> Lgtm minus one nit, it would be nice to align the lines from the hyphen on.

they are actually aligned on output, since the carets do not display. Since people changing it might not know or notice that, it's likely to bit rot...

**Comment by jashook** [comment]
Gotcha makes sense

File: tests/runtest.cmd [hunk 1]
```diff
@@ -432,29 +432,36 @@ exit /b 0
...
+echo longgc                    - Run the long-running GC tests
+echo gcsimulator               - Run the GC Simulator tests
+echo gcstresslevel ^<n^>         - Runs the tests with COMPlus_GCStress=n. n=0 means no GC Stress. Otherwise, n is a bitmask of the following:
+echo                               1: GC on all allocs and 'easy' places
+echo                               2: GC on transitions to preemptive GC
+echo                               4: GC on every allowable JITed instruction
+echo                               8: GC on every allowable NGEN instruction
+echo                              16: GC only on a unique stack trace
+echo msbuildargs ^<args...^>     - Pass all subsequent args directly to msbuild invocations.
+echo ^<CORE_ROOT^>               - If specified, last, this is the path to the runtime to test.
```
Comment (RussKeldorph) [inline tests/runtest.cmd]: I would write `Path to the runtime to test (if specified)`.

File: tests/runtest.cmd [hunk 2]
```diff
@@ -432,29 +432,36 @@ exit /b 0
 :Usage
 echo.
 echo Usage:
-echo   %0 BuildArch BuildType [TestEnv TEST_ENV_SCRIPT] [VSVersion] CORE_ROOT
+echo   %0 [options] [CORE_ROOT]
```
Comment (RussKeldorph) [inline tests/runtest.cmd]: My understanding of the typical cmd line convention is that `CORE_ROOT` should be in `< >` to indicate it is the name of an argument, not the value, corresponding to below.

**Pull request merged**